### ISSUE TITLE
[now dev] Installed packages in `vendor`do not exists

### ISSUE THREAD

**Issue by ties-v** [juicyfx/now-php#12]
At this moment my project has a `.nowignore` file to prevent the composer `vendor` directory from uploading to now.

When I run this same project with `now dev` I get an error because the `vendor` directory is not found. This makes sense because it is ignored.

This should however be resolved by either:
* Also run composer install in `now dev`, but this seems to make it really slow.
* Also copy the `vendor` directory during a development run. But how can that be done?

**Comment by f3l1x** [comment]
Yep, I've pointed that out also to ZEIT team and we need to colaborate on it.

I picture it like this:

- now
  - composer deps are installed if composer.json is in place

- now dev
  - composer deps are installed if composer.json is in place only once, for the first time
  - we need to track if composer.json or composer.lock is changed and re-install composer deps

There is way I think, but it takes so much time to do it, because you have to publish pkg on npmjs to use it in now dev, thus we need to cooperate with ZEIT in this thing.
